Conan.VisualStudio/source.extension.vsixmanifest

```diff
@@ -12,6 +12,7 @@
     </Metadata>
     <Installation>
         <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,16.1]" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,16.1]" />
```

Comment:
This is not needed, `Pro` includes `Community`

Reply:
okay, what about `Enterprise`?

Reply:
According to this it should be enough: https://msdn.microsoft.com/es-es/en-en/magazine/mt493251.aspx

> el valor Microsoft.Visual­Studio.Pro está dirigido a la edición Professional y superior, como las ediciones Premium, Ultimate, Enterprise y demás. Tenga en cuenta que también está dirigida a la edición Community
english: 
> the Microsoft.VisualStudio.Pro value is intended for Professional and higher editions, such as Premium, Ultimate, Enterprise, and others. Please note that it is also aimed at the Community edition.